packages/@lwc/template-compiler/src/codegen/index.ts

```diff
@@ -53,6 +51,8 @@ import {
     isSvgUseHref,
 } from '../parser/attribute';
 
+import State from '../state';
+import { isCustomRendererHookRequired } from '../shared/renderer-hooks';
```

Comment:
Group parent import together.

```suggestion
} from '../parser/attribute';
import State from '../state';
import { isCustomRendererHookRequired } from '../shared/renderer-hooks';

```